wdl-analysis/src/lib.rs

```diff
@@ -20,5 +20,90 @@ mod rules;
 pub mod stdlib;
 pub mod types;
 
+use std::time::Duration;
+
 pub use analyzer::*;
+use anyhow::bail;
+use indicatif::ProgressBar;
+use indicatif::ProgressStyle;
 pub use rules::*;
+use tokio::fs;
+use url::Url;
+
+/// The delay in showing the progress bar.
+const PROGRESS_BAR_DELAY: Duration = Duration::from_secs(2);
```

Comment:
I'd probably do every second here.

Reply:
Maybe this comment should be clarified. This is adding a 2 second delay _before_ the progress bar appears.

I found that we were analyzing files/directories so quickly that the inclusion of the progress bar actually looked pretty crummy because it would pop up and then instantly be dropped. So this essentially prevents the progress bar from showing up unless the analysis is taking a long time (most likely due to a slow network connection. That's really the only bottleneck here).

Reply:
Okay, I see. That sounds good then.

Reply:
Perhaps then `PROGRESS_BAR_DELAY_BEFORE_RENDER`.